io.catenax.pcf/4.0.0/Pcf.ttl

```diff
@@ -0,0 +1,896 @@
+
+:PrecedingPfId a samm:Entity ;
+   samm:preferredName "Preceding PCF (Product Carbon Footprint) Identifier Entity"@en ;
+   samm:description "Entity for defining a preceding PCF (Product Carbon Footprint) identifier entity as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:properties ( :id ) .
+
+:CompanyId a samm:Entity ;
+   samm:preferredName "Company Identifier"@en ;
+   samm:description "Company identifier in context of a product (carbon) footprint which must be declared as URN (Uniform Resource Name) as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:properties ( :companyId ) .
+
+:ProductId a samm:Entity ;
+   samm:preferredName "Product Identifier"@en ;
+   samm:description "Identifier of a product in context of a product (carbon) footprint as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:properties ( :productId ) .
+
+:declaredUnit a samm:Property ;
+   samm:preferredName "Unit of measurement"@en ;
+   samm:description "Unit of analysis of a product in context of the PCF (product carbon footprint) as specified in the Catena-X PCF Rulebook in accordance with the technical specifications for PCF Data Exchange (Version 2.0.0) from the WBCSD (World Business Council for Sustainable Development)/ PACT initiative. In Catena-X list of valid units includes \"piece\"."@en ;
+   samm:see <https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/> ;
+   samm:characteristic :DeclaredUnitCharacteristic ;
+   samm:exampleValue "kilogram" .
```

Comment:
type reference missing?

Reply:
Like in the WBCSD/ PACT specification property is defined as enumeration of strings. No need to adapt or change.